packages/fx-core/src/plugins/resource/sql/plugin.ts

```diff
@@ -42,42 +42,36 @@ import { Bicep, ConstantString } from "../../../common/constants";
 import { ScaffoldArmTemplateResult } from "../../../common/armInterface";
 import * as fs from "fs-extra";
 import { getArmOutput } from "../utils4v2";
-import { isArmSupportEnabled } from "../../../common";
+import { getResourceGroupNameFromResourceId, isArmSupportEnabled } from "../../../common";
 import { IdentityArmOutput } from "../identity/constants";
 
 export class SqlPluginImpl {
   config: SqlConfig = new SqlConfig();
 
-  async init(ctx: PluginContext) {
-    ContextUtils.init(ctx);
-    this.config.azureSubscriptionId = ContextUtils.getConfigString(
+  async loadConfig(ctx: PluginContext) {
+    this.config.azureSubscriptionId = ContextUtils.getConfig<string>(
```

Comment:
It's possible that user provision the resource to another subscription, we also need to read subscriptionid from resource id

Reply:
It is fixed. Thanks.